docs/production/management-commands.md

```diff
@@ -82,6 +82,15 @@ In [1]: user_profile = get_user_profile_by_email("email@example.com")
 In [2]: do_change_user_delivery_email(user_profile, "new_email@example.com")
 ```
 
+you can do the following to get your organization's realm object if you have
```

Comment:
You forgot to capitalize "you".

Reply:
Ah yes! thanks for pointing that out! updated the PR :)